coursier.bzl

```diff
@@ -466,16 +467,24 @@ def _deduplicate_artifacts(dep_tree):
 # Get the path to the cache directory containing Coursier-downloaded artifacts.
 #
 # This method is public for testing.
-def get_coursier_cache_or_default(env_dict, use_unsafe_shared_cache):
-    location = env_dict.get("COURSIER_CACHE")
-
-    # If we're not using the unsafe shared cache, or if COURSIER_CACHE is not defined,
-    # use 'external/<this repo>/v1/'. 'v1/' is the current version of the Coursier cache.
-    if not use_unsafe_shared_cache or location == None:
+def get_coursier_cache_or_default(os_env, os_name, use_unsafe_shared_cache):
+    # If we're not using the unsafe shared cache use 'external/<this repo>/v1/'.
+    # 'v1' is the current version of the Coursier cache.
+    if not use_unsafe_shared_cache:
         return "v1"
-    else:
+
+    coursier_cache_env_var = os_env.get("COURSIER_CACHE")
+    if coursier_cache_env_var:
         # This is an absolute path.
-        return location
+        return coursier_cache_env_var
+
+    # locations from https://get-coursier.io/docs/1.1.0-M14-1/cache.html#default-location
```

Comment:
The current version is https://get-coursier.io/docs/2.0.0-RC5-3/cache.html#default-location

Reply:
updated